### ISSUE TITLE
We really doesn't need to create custom markup extension 

### ISSUE THREAD

**Issue by IbrahimElshafey** [MicrosoftDocs/xamarin-docs#256]
We really don't need to create custom markup extension to localize XAML strings I use [x:Static Markup Extension](https://docs.microsoft.com/en-us/dotnet/framework/xaml-services/x-static-markup-extension) and it works fine.
Something like that `Text={x:static Appresources.TitleLabel}`

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: af66394d-81c5-0c2e-1682-a2bb7eeb3c3e
* Version Independent ID: 27542a45-2f25-4042-cf48-b9517025635c
* Content: [String and Image Localization - Xamarin](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/localization/text?tabs=vswin#localizing-xaml)
* Content Source: [docs/xamarin-forms/app-fundamentals/localization/text.md](https://github.com/MicrosoftDocs/xamarin-docs/blob/live/docs/xamarin-forms/app-fundamentals/localization/text.md)
* Service: **xamarin**
* GitHub Login: @davidbritch
* Microsoft Alias: **dabritch**

**Comment by davidbritch** [comment]
Hi @IbrahimElshafey 

Thanks for the tip. I've created an issue in our backlog for this item. Thanks!

**Comment by AntRemo** [comment]
@IbrahimElshafey I too have been using the `Text={x:static Appresources.TitleLabel}` as well.  👍

**Comment by nhwilly** [comment]
@davidbritch FYI, I implemented all the recommended code in the docs and then discovered that this simpler method works for me, too.  I see that you closed the issue, but what does that mean exactly?  Is this a good idea?  

Which practice is recommended?  And why?

**Comment by AntRemo** [comment]
@nhwilly all great questions. I really don't know why there is a need for the extra overhead the doc suggests.

**Comment by nhwilly** [comment]
I am really struggling with the UWP version. Android and iOS work fine, but the UWP version refuses to see the neutral file values. 

Yes, I tried with and without modifying AssemblyInfo.cs and cleaning/rebuilding /deleting bin dirs.

Only works when I create and appresources.en.resx file.

**Comment by nhwilly** [comment]
*an AppResources.en.resx file.

I'll try again tomorrow following the guy I saw on Ch9.

**Comment by davidbritch** [comment]
@nhwily

I've logged an issue in our backlog so I can investigate further and update the doc/sample if required.

**Comment by nhwilly** [comment]
Ok, I talk too much, but assuming more information is preferred to less, here's what I've discovered over the last couple of days.  

**Default language**
The instructions [here](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/localization/text?tabs=vswin#assemblyinfocs) about `AssemblyInfo.cs` didn't work for me.  That might be because they tell me to make those changes to that file in the shared project and no such file exists in my project.  As a result, I thought maybe the author meant making the changes in the UWP file (which does exist) since that's where my problems were anyway.  Nope.  That did nothing.
However, making this change to the shared project `.CSPROJ` did:
```
<PropertyGroup>
  <TargetFramework>netstandard2.0</TargetFramework>
  <NeutralLanguage>en-US</NeutralLanguage>
  <ApplicationIcon />
  <Win32Resource />
</PropertyGroup>
```
I did not try it with just "en" set.  Maybe later.
I stumbled across this solution while researching the [Multilingual App Toolkit](https://docs.microsoft.com/en-us/windows/uwp/design/globalizing/use-mat) - which is completely awesome by the way and will be the primary way I attack this feature in the end.

**Binding**
The XAML `{x:Static resources.AppResources.StringYouWanted}"` works.  I tried the plugin, but less is preferred to more, so if this binding works, then I'm super happy.  It works really well on the Android and iOS simulators and after changing the default setting above, it worked on UWP, too.

As a suggestion for the docs, you might recommend that they clean the project as the resource files seem to hang around, especially for UWP.  I just had to delete an entire page of what I thought was a bug until I figured I'd clean the project and try once more.  Live and learn. :)

I know from the article that one of the benefits of having all the plugin code is to be able to accommodate languages that aren't available and other app specific edge cases.  But for more of us just trying to add a few common languages and willing to go to market without Klingon, a paragraph on static binding seems like something useful.

HTH
